objective-c/HelloWorld/HelloWorld/AppDelegate.m

```diff
@@ -0,0 +1,45 @@
+//
+//  AppDelegate.m
+//  HelloWorld
+//
+//  Created by Michael Lumish on 6/11/15.
+//  Copyright (c) 2015 Google. All rights reserved.
+//
+
+#import "AppDelegate.h"
+
+@interface AppDelegate ()
+
+@end
```

Comment:
You need to define the implementation of the class. You _could_ do both declaring the interface and defining the implementation in main.m, I presume. But you might as well just hide the three files inside a subdir.